### ISSUE TITLE
appsody operator uninstall of second operator in separate namespace fails

### ISSUE THREAD

**Issue by tnixa** [appsody/appsody#226]
**Describe the bug**
When you have multiple operators running in different namespaces you will not be able to delete anything other than the first one

**To Reproduce**
1. run docker desktop and its k8s
2. `mkdir nodejs`
3. `cd nodejs`
4. `appsody init nodejs`
5. `appsody deploy`
6. `appsody deploy --namespace test`
7. `appsody operator uninstall`
8. `appsody operator uninstall --namespace test`

**Expected behavior**
Expecting this to work without errors

**Actual behaviour**
```
supra:nodejs tnixa$ appsody operator uninstall --namespace test
Attempting to delete resource from Kubernetes...
Running command: kubectl[delete -f /Users/tnixa/.appsody/deploy/appsody-app-crd.yaml --namespace test]
[Error] warning: deleting cluster-scoped resources, not scoped to the provided namespace
Error from server (NotFound): error when deleting "/Users/tnixa/.appsody/deploy/appsody-app-crd.yaml": customresourcedefinitions.apiextensions.k8s.io "appsodyapplications.appsody.dev" not found
[Error] kubectl delete failed: exit status 1
[Error] exit status 1
```

**Environment Details (please complete the following information):**
 - OS: mac

**If applicable please specify:**
 - CLI version: 0.3.0
 - Stack you are using: nodejs

**Comment by chilanti** [comment]
Should be fixed by PR #252

**Comment by kewegner** [comment]
This was fixed by #252
